src/components/dashboard/PendingTxs/PendingTxsList.tsx

```diff
@@ -43,12 +42,22 @@ const PendingTxsList = ({ size = 4 }: { size?: number }): ReactElement | null =>
   const router = useRouter()
   const url = `${AppRoutes.transactions.queue}?safe=${router.query.safe}`
 
-  const queuedTxns: Transaction[] = (page?.results || []).filter(isTransactionListItem)
+  const queuedTxns = useMemo(() => {
+    return groupConflictingTxs(page?.results || [])
+      .map((group) => {
+        if (Array.isArray(group)) {
+          const latestTx = group.reduce((acc, tx) => {
+            return tx.transaction.timestamp > acc.transaction.timestamp ? tx : acc
```

Comment:
I would suggest naming them `prev` and `curr` or something similar ([mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#syntax) uses `previousValue` and `currentValue` in their docs) since we are not accumulating a value in this case.